cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -120,8 +123,8 @@ public TestCommand() {
             "dependencies.")
     private Boolean offline;
 
-    @CommandLine.Parameters (arity = "0..1")
-    private final Path projectPath;
+    @CommandLine.Parameters(description = "Program arguments")
```

Comment:
`description` starts with upper case in some places and lower case in other places. can we follow a consistent format here?

Reply:
This same concern was noted earlier and since these are not directly shown to the user it was decided that changing all of them wouldn't be necessary